app/models/rdv.rb

```diff
@@ -129,6 +129,10 @@ def user_for_home_rdv
     [responsibles, users].flatten.select(&:address).first || users.first
   end
 
+  def for_service_social?
+    agents.select { |a| a.service.service_social? }.any?
```

Comment:
actually sorry for the last 2 comments, re-reading I don't think this method is correct. Rdv's service is not defined by the associated agents service's but rather by the motif, no ? 
So I think the RDV should delegate its `service` methods to the `motif` and the `motif` to the `service`.